### PR TITLE
Add config file examples to sensuctl reference.

### LINKED ISSUE #1158: Provide examples for sensuctl config files
In the sensuctl reference, provide examples for the sensuctl config files at cat .config/sensu/sensuctl/profile and cat .config/sensu/sensuctl/cluster.

```
cat .config/sensu/sensuctl/profile
{
  "format": "tabular",
  "namespace": "demo"
}
```

Depends on 5.2

### PR DESCRIPTION
**Pull request by asachs01**
## Description
Adds config file examples to the sensuctl reference doc. 

## Motivation and Context
Closes #1158

### CODE REVIEW

File: content/sensu-go/5.2/sensuctl/reference.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ menu:
 - [Managing resources](#managing-resources)
 - [Time formats](#time-formats)
 - [Shell auto-completion](#shell-auto-completion)
+- [Config files](#)
```
Comment (rgeniesse) [inline content/sensu-go/5.2/sensuctl/reference.md]: Should this have more after the `#`? Maybe `configuration-files`?
```suggestion
- [Config files](#configuration-files)
```

File: content/sensu-go/5.2/sensuctl/reference.md [hunk 2]
```diff
@@ -448,6 +449,33 @@ asset       completion  entity      handler
 create  delete  import  list
 {{< /highlight >}}
 
+## Configuration files
+
+When configuring sensuctl, configuration files will be created that contain information for connecting to your Sensu Go deployment. You can find them at `$HOME/.config/sensu/sensuctl/profile` and `$HOME/.config/sensu/sensuctl/profile`. Here are two examples:
```
Comment (apaskulin) [inline content/sensu-go/5.2/sensuctl/reference.md]:
```suggestion
During configuration, sensuctl creates configuration files that contain information for connecting to your Sensu Go deployment. You can find them at `$HOME/.config/sensu/sensuctl/profile` and `$HOME/.config/sensu/sensuctl/profile`. For example:
```

File: content/sensu-go/5.2/sensuctl/reference.md (hunk 1 above)
Comment (asachs01) [reply content/sensu-go/5.2/sensuctl/reference.md]: Right, I hadn't determined what the link would be when I was writing. Will update with this.

**Comment by asachs01** [comment]
@rgeniesse can you check me again? Should be g2g now.

**Review by apaskulin — APPROVED** [review]
🎉

**Review by rgeniesse — APPROVED** [review]
looks good now. 👍

**Pull request merged**